meson.build

```diff
@@ -1,46 +1,49 @@
-project(
+# Project name, programming language and version
+project (
     'com.github.peteruithoven.resizer',
-    ['vala', 'c'],
-    meson_version : '>= 0.47',
```

Comment:
Why not define the min. required meson?

Reply:
It definitely should be defined!

Meson can refuse to run on older versions, while printing an informative error message telling you that your version of Meson is too old. Additionally, it also prints informative warnings when you use features too new for the minimum required version to handle.

Reply:
I can add it back but even elementary apps does not include this. I assume developers should check that their development tools are up to date before trying to build the app. But I can add it back if you prefer.

Reply:
Why make life harder, when you could make life easier instead? :)